components/sensor/ld2410.rst

```diff
@@ -0,0 +1,205 @@
+LD2410 Sensor
+=============
+
+.. seo::
+    :description: Instructions for setting up LD2410 sensors.
+    :image: ld2410.jpg
+
+Component/Hub
+-------------
+.. _ld2410-component:
+
+The ``ld2410`` sensor platform allows you to use HI-LINK LD2410 motion and presence sensor.
+(`datasheet and user manual <https://drive.google.com/drive/folders/1p4dhbEJA3YubyIjIIC7wwVsSo8x29Fq-?spm=a2g0o.detail.1000023.17.93465697yFwVxH>`__) with ESPHome.
+The :ref:`UART <uart>` is required to be set up in your configuration for this sensor to work.
```

Comment:
I think it might be worth adding a note here or nearby about the need for hardware serial; I got caught out when trying to use software serial pins on an ESP8266, which don't support the required 256000 baud rate.

Maybe something like this:
```suggestion
The :ref:`UART <uart>` is required to be set up in your configuration for this sensor to work.
Use of hardware UART pins is highly recommended, in order to support the out-of-the-box 256000 baud rate of the LD2410 sensor.
```